packages/common/src/compiler/createHosts.ts

```diff
@@ -65,7 +68,8 @@ export function createHosts(options: CreateHostsOptions) {
                 );
             }
         },
-        useCaseSensitiveFileNames: () => true,
+        isKnownTypesPackageName: isKnownTypesPackageName || returnTrue,
```

Comment:
I seem to not be able to push commits to this PR, so I'm going to merge this as-is, but afterwards I'm going to remove this `|| returnTrue` since I'd rather have undefined be the default as is done in the compiler api.